test/BF16/mlp-single-layer-bf16.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: tpp-opt %s -linalg-ext-to-loops -convert-linalg-to-loops -convert-tpp-to-xsmm -convert-xsmm-to-func -convert-vector-to-scf -convert-scf-to-cf -convert-vector-to-llvm -convert-func-to-llvm -convert-memref-to-llvm -canonicalize -sparse-compiler|\
+// RUN: tpp-opt %s -linalg-ext-to-loops -convert-linalg-to-loops -convert-tpp-to-xsmm -convert-xsmm-to-func -convert-vector-to-scf -convert-scf-to-cf -sparse-compiler |\
```

Comment:
Can we drop `sparse-compiler`?

Reply:
Maybe? I don't know why it was chosen here... But these aren't the only examples, so I'm guessing these were copy from some others.

Reply:
It was probably missing a conversion. We should remove it, but we can do it later.

Reply:
#87